package.json

```diff
@@ -19,56 +19,57 @@
     "prepare": "cd .. && husky install ui/.husky"
   },
   "dependencies": {
-    "@apollo/client": "^3.7.0",
+    "@apollo/client": "^3.7.3",
     "@hookform/resolvers": "^2.9.10",
-    "apollo-link-rest": "^0.9.0-rc.2",
+    "@sentry/browser": "^7.28.1",
+    "apollo-link-rest": "^0.9.0",
     "classnames": "^2.3.2",
     "date-fns": "^2.29.3",
-    "framer-motion": "^7.6.9",
+    "framer-motion": "^8.0.2",
```

Comment:
we have ot keep framer-motion in sync w/ rdk/reaviz/realayers so we don't end up w/ 2 versions.